lib/cmd/fh3/admin/environments/update.js

```diff
@@ -1,13 +1,14 @@
 /* globals i18n */
 module.exports = {
   'desc' : i18n._('Update an environments.'),
-  'examples' : [{ cmd : 'fhc admin environments update --id=<environment id> --label=<label> --target=<mbaasTargetId>', desc : i18n._('Update an environment by id')}],
+  'examples' : [{ cmd : 'fhc admin environments update --id=<environment id> --label=<label> --target=<mbaasTargetId> --token=<token>', desc : i18n._('Update an environment by id')}],
   'demand' : ['id'],
   'alias' : {},
   'describe' : {
     'id' : i18n._('Some unique identifier for your environment'),
     'label' : i18n._('A label describing your environment'),
-    'target' : i18n._('unique identifier for an existing MBaaS Target')
+    'target' : i18n._('unique identifier for an existing MBaaS Target'),
+    'token': i18n._('Openshift token required to create the environment project')
```

Comment:
Also not sure how its handled but the token should be a required param for the env creation if the environment is an openshift3 one